prsa/signal_preprocessing.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+from scipy.ndimage import label
+from scipy.signal import find_peaks
+from scipy.stats import zscore
+
+def normalize_time_series(time_series: np.ndarray) -> np.ndarray:
```

Comment:
w teorii normalizacja to przekształcenie na rozkład normalny standardowy -> odjęcie średniej i podzielenie przez odchylenie standardowe.
skalowanie do 0-1 to standaryzacja

w praktyce jest to ekstremalnie pomieszane i używane przypadkowo, więc pozostawiam do decyzji.